pkg/autoscaler/multiscaler.go

```diff
@@ -35,6 +35,11 @@ const (
 	// seconds while an http request is taking the full timeout of 5
 	// second.
 	scaleBufferSize = 10
+
+	// samplesPerSecond is the times to scrape metrics per second across all pods
+	// of a revision.
+	// TODO(yanweiguo): tuning this value. To be based on pod population?
+	samplesPerSecond = 3
```

Comment:
I'd calculate a tick scrapeTickInterval here directly based off of this value so tests can share it without needing to recalculate.

Reply:
Done.